src/libraries/System.Net.Security/src/System/Net/NegotiateAuthenticationPal.Unix.cs

```diff
@@ -13,16 +13,20 @@
 using System.Security.Authentication.ExtendedProtection;
 using System.Security.Principal;
 using System.Text;
+using System.Text.RegularExpressions;
 using Microsoft.Win32.SafeHandles;
 
 namespace System.Net
 {
     internal partial class NegotiateAuthenticationPal
     {
+        private static readonly Lazy<bool> _hasSystemNetSecurityNative = new Lazy<bool>(CheckHasSystemNetSecurityNative);
+        internal static bool HasSystemNetSecurityNative => _hasSystemNetSecurityNative.Value;
         private static bool UseManagedNtlm { get; } =
             AppContext.TryGetSwitch("System.Net.Security.UseManagedNtlm", out bool useManagedNtlm) ?
             useManagedNtlm :
-            OperatingSystem.IsMacOS() || OperatingSystem.IsIOS() || OperatingSystem.IsMacCatalyst();
+            OperatingSystem.IsMacOS() || OperatingSystem.IsIOS() || OperatingSystem.IsMacCatalyst() ||
+            (OperatingSystem.IsLinux() && Regex.IsMatch(RuntimeInformation.RuntimeIdentifier, "^linux-bionic(-.*)?$", RegexOptions.CultureInvariant | RegexOptions.NonBacktracking | RegexOptions.ExplicitCapture));
```

Comment:
is there? I did not know, I will change it then

Reply:
It's documented:

> .NET 8 and later versions: This property returns a string that identifies the operating system and processor architecture for which the runtime was built. For example, the value on Windows 11 is win-x64, and on Ubuntu 20.04, it's linux-x64.

(for .NET <=7 it could return non-portable RIDs but we don't care about that)